frontend/src/components/Forms/MineRecordForm.js

```diff
@@ -49,24 +48,31 @@ const propTypes = {
   conditionalCommodityOptions: PropTypes.objectOf(CustomPropTypes.options).isRequired,
   currentMineTypes: PropTypes.arrayOf(CustomPropTypes.mineTypes),
   submitting: PropTypes.bool.isRequired,
+  isCreated: PropTypes.bool,
 };
 
 const defaultProps = {
   title: "",
   currentMineTypes: [],
   mine_types: [],
-  mineStatus: [],
+  isCreated: false,
 };
```

Comment:
I think we can do better with this constant, I know I initially named it, my bad. 
I think `isNewRecord` or `isNewMine` or something to that effect would be better.